src/rust/engine/process_execution/src/lib.rs

```diff
@@ -260,6 +260,8 @@ pub struct Process {
   pub target_platform: PlatformConstraint,
 
   pub is_nailgunnable: bool,
+
+  pub cache_failures: bool,
```

Comment:
It's already the case that we don't store failed results in this cache, right? We don't execute `command_runner.store()` unless `result.exit_code` is 0? I'm not sure where it would make sense to check the value of `cache_failures` in this codepath.

Reply:
The value is a boolean... the implication is that if someone set `cache_failures: true`, we would cache the failure. And that code would need to change to do that.